### ISSUE TITLE
Member profile and links to their own professional social media accounts

### ISSUE THREAD

**Issue by SimonJinaphant** [UBC-Snowbots/UBC-Snowbots.github.io#5]
Similar to the old site: Have a more indepth profile for each member and possibly link to their Linkedin, Github etc..

Suggested by Arjun

**Comment by Dulluhan** [comment]
I was thinking the same, I'm going to with with github for software. For other electrical and mech probably linkedin. Do you think just a href to the block is ok?

**Comment by SimonJinaphant** [comment]
Doesn't font-awesome have those logo in bootstrap? I know they exist for the one I used in my personal site. We could have some smaller (I mean really small like the icons to the side of this page---->) on the each of the profile. I don't really want to restrict one person to having one thing based on their sub-team

EDIT: I've uploaded a demo on the roster page, scroll down to the software section. Personally I think it only looks good when you have at-least 2-3 icons, anything less than that looks kinda awkward.
Also if only one person has an icon it will create an awkward vertical gap between the thumbnail.
